### ISSUE TITLE
GitHub (Local) hides GitHub identity provider

### ISSUE THREAD

**Issue by nngo** [Apicurio/apicurio-keycloak-extensions#3]
I have built and deploy this plugin under Keycloak 9.0.3.
When I try to add a new identity provider, I only see "GitHub (Local)", the "GitHub" identify provider is not in the list.

Could it be because LocalGitHubIdentityProviderFactory.getId() returns "github"? should that be changed to return "github-enterprise" or "github-local" or something like that.

Another thing, should LocalGitHubIdentityProviderFactory.getName() return "GitHub (Enterprise)" instead of "Git (Local)"?

**Comment by EricWittmann** [comment]
These are excellent suggestions.  It's been awhile since I've looked at the GitHub integration, but I recall that value returned by "getId()" to be important on the Apicurio side of things.  If that is changed in the extension then it might not be usable from Apicurio Studio.  If you want to support GH Enterprise and GitHub Online simultaneously in the same Apicurio Studio server, then we will need to make some additional changes in the studio repo.

**Comment by EricWittmann** [comment]
That said - the name change we can probably make immediately. :)

**Comment by nngo** [comment]
you are right, I tried switch the id and it causes issue in creating this identity provider since it requires theme/base/admin/resources/partials/realm-identity-provider-<provider-id>*.html in the keycloak installation dir. I did successfully workaround that, but it requires more steps during the installation procedure... not sure if it is worth it, so will close this ticket. I think a better thing is to update the GitHubIdentityProvider in the keycloak code base to support our desired functionality.
